### DISCUSSION TITLE
User account limit

### DISCUSSION THREAD

**Discussion by DaInfLoop** [ppy/osu#28685] [Q&A] [answered]
I wanted to try create an account for a project I'm working on, but got a warning in-game that I can't have more than one account.

My idea for this project was to have some sort of verification for Slack (and also since Slack doesn't have native rich presence).

I would use a new account (and not play with it at ALL) to join IRC and then have users message the bot a code given to them on Slack either on IRC or in-game chat since they're linked.

I would like some sort of verification that either this would be okay or not. Thanks in advance.

**Comment by Joehuu** [comment, marked as answer]
I think https://osu.ppy.sh/wiki/en/Bot_account is what you're looking for.

**Reply by DaInfLoop** [reply]
Thanks! I was mainly seeing if there was I way I could run it not from my main account, but I guess I'll have to for now.
